tests/foreman/cli/test_contenthost.py

```diff
@@ -567,115 +559,38 @@ def setUp(self):
 
```

Comment:
If code is used in the beginning of each test of the class, than it _should_ be moved to setUp(), that what setUp is made for :)
What are you trying to suggest is probably not ```setUp()```, but ```setUpClass()```, isn't it?
In this case, code can be splitted to something like that:
```python
class TestContentHostKatelloAgent(CLITestCase):

    def setUpClass():
    # Create org, env, CV, AK

    def setUp():
    # Create VM, install katello-agent

    def tearDown():
    # Destroy VM

    def test_1():
    # Testing here
```
This will allow us to use 1 org, env and AK for all the tests and we won't duplicate VM code inside each test. Did I understand you correctly?